tests/e2e/support/commands.js

```diff
@@ -240,6 +303,39 @@ Cypress.Commands.add('playPointsOpponent', (card) => {
     });
 });
 
+/**
+ * @param card {suit: number, rank: number}
+ */
+Cypress.Commands.add('playPointsSpectator', (card, pNum) => {
```

Comment:
@seriouslysean We talked about wrapping this into a separate command -- atm since points is the only move the tests require I think it's okay to leave this as is and to revisit it as a part of the follow up issue where we add test cases for a wider range of moves